remote/src/main/resources/reference.conf

```diff
@@ -492,7 +492,7 @@ pekko {
         applied-adapters = []
 
         # The default remote server port clients should connect to.
-        # Default is 7355 (PEKK on a telephone keypad), use 0 if you want a random available port
```

Comment:
7355 is PEKK - 4 digits, 4 letters - so can you undo this change?

Reply:
Thanks, I forgot the previous discussion of pekko default listener port.